deltachat-contact-utils/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "deltachat-contact-utils"
+version = "0.1.0"
+edition = "2021"
+description = "Contact-related tools, like parsing vcards and sanitizing name and address"
+license = "MPL-2.0"
+# TODO maybe it should be called "deltachat-text-utils" or similar?
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+once_cell = { workspace = true }
+regex = { workspace = true }
+rusqlite = { workspace = true } # Needed in order to `impl rusqlite::types::ToSql for EmailAddress`. Could easily be put behind a feature.
```

Comment:
If it can be grepped, mybe the comment is not needed

Reply:
Even though it's possible to just grep why you need dependencies, I think it's good practice to include the justification if it's not obvious. Not something that's important for me or anything, but since you wrote "maybe", I'm going to assume that it's also not important for you